src/test-utils/consoleErrorMock.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable flowtype/require-valid-file-annotation, no-console */
```

Comment:
Thats a good question. I couldn't find any documentation for this on Flow. Would love some of your insight into this?